packages/apps/kadena-docs/src/components/Layout/components/SideMenu/MenuCard.tsx

```diff
@@ -0,0 +1,25 @@
+import { StyledSection } from './styles';
+
+import React, { FC, ReactNode } from 'react';
+
+export interface IMenuCardProps {
+  children?: ReactNode;
+  active: number;
+  idx: number;
+  ref?: React.ForwardedRef<HTMLDivElement>;
+}
+
+export const MenuCard: FC<IMenuCardProps> = React.forwardRef(
+  ({ children, active, idx = 0 }, ref) => {
+    return (
+      <StyledSection
+        animateLeft2Right={idx === 0}
```

Comment:
nitpick: I'm not sure what this is, but assuming it doesn't have any other animations styles, I would suggest naming it `animateIn` or something that's less specific since the person using the component doesn't need to know how it is animating